README.md

```diff
@@ -1,7 +1,7 @@
 msa
 =====
 
-BioJS: Multiple Sequence Alignment (MSA) Viewer
+Multiple Sequence Alignment Viewer - the MSAViewer - a BioJS component.
```

Comment:
Okay but we should remove the term BioJS soon - it's not something I am proud of :/